app/views/characters/new.html.erb

```diff
@@ -0,0 +1,8 @@
+<h2> Create a new person to die </h2>
+<%= render "form" %>
+
+<%= form_for @character, method: :create do |f| %>
```

Comment:
No need to specify `method: :create` if `@character` is a new one. Also, you would pass `:post` into `method:`, not `:create`